scripts/setup_quickstart_framework.sh

```diff
@@ -25,7 +25,7 @@ cd quickstart-ios/"${SAMPLE}"
 # the project for framework-based testing.
 if [[ "${SAMPLE}" == "FirebaseAI" ]]; then
   # For the FirebaseAI quickstart, we only want to remove the Firebase deps.
-  "${REPO}"/scripts/remove_spm_dependencies.rb "${SAMPLE}Example.xcodeproj" FirebaseAI
+  "${REPO}"/scripts/remove_spm_dependencies.rb "${SAMPLE}Example.xcodeproj" FirebaseAILogic
```

Comment:
Thanks, Nick! This will help with being able to clean up the `#if canImport(FirebaseAILogic)` bits in the quickstart too.